### PR TITLE
Reviewed and updated German translations

### PR DESCRIPTION
**Pull request by jensens**
- new ones,
- improvements,
- fixed typos.

### CODE REVIEW

File: src/collective/fingerpointing/locales/de/LC_MESSAGES/collective.fingerpointing.po [hunk 1]
```diff
@@ -49,39 +50,39 @@ msgstr "Finger Pointing"
...
 
 #: collective/fingerpointing/interfaces.py:38
 msgid "Log registry events like records being modified."
-msgstr "Protokolliert Änderungen der Konfigurationseinträge (plone.app.registry)."
+msgstr "Protokolliert Änderungen der Konfigurationseinträge (Registry)."
 
 #: collective/fingerpointing/interfaces.py:32
 msgid "Log workflow transitions like object publishing."
-msgstr ""
+msgstr "Protokolliert Arbeitsablauf Übergänge, wie z.B. das Publizieren von Inhalten."
```
Comment (frisi) [inline src/collective/fingerpointing/locales/de/LC_MESSAGES/collective.fingerpointing.po]: statt "publizieren" evtl transition titel wie "veröffentlichen / zurückziehen" verwenden

File: src/collective/fingerpointing/locales/de/LC_MESSAGES/collective.fingerpointing.po [hunk 2]
```diff
@@ -49,39 +50,39 @@ msgstr "Finger Pointing"
...
 
 #: collective/fingerpointing/browser/logview.pt:26
 msgid "No audit log file specified; audit log view is disabled."
-msgstr ""
+msgstr "Keine Protokoll-Datei angegeben, die Aktionsprotokoll Ansicht ist deaktiviert."
 
 #: collective/fingerpointing/controlpanel.py:13
 msgid "Settings for the collective.fingerpointing package"
-msgstr "Einstellungen für collective.fingerpointing"
+msgstr "Einstellungen für das collective.fingerpointing Paket"
```
Comment (frisi) [inline src/collective/fingerpointing/locales/de/LC_MESSAGES/collective.fingerpointing.po]: having a look at prefs_install_products_form it looks like "Erweiterung" is more common
in addition there are several profiles w/o package or addon in their description.
"Optionale Aktualisierungsunterstützung für plone.session"
i'd leave this one as it was, or go with
"Einstelllungen für die Erweiterung collective.fingerpointing"

File: src/collective/fingerpointing/locales/de/LC_MESSAGES/collective.fingerpointing.po [hunk 3]
```diff
@@ -49,39 +50,39 @@ msgstr "Finger Pointing"
...
 
 #: collective/fingerpointing/interfaces.py:25
 msgid "Log content type life cycle events like object creation, modification and removal."
-msgstr "Protokolliert den Ereignisse im Lebenszyklus von Inhalten (Erzeugen, Bearbeiten, Löschen)"
+msgstr "Protokolliert die Ereignisse im Lebenszyklus von Inhalten, wie Erstellen, Bearbeiten und Löschen."
 
 #: collective/fingerpointing/interfaces.py:38
 msgid "Log registry events like records being modified."
-msgstr "Protokolliert Änderungen der Konfigurationseinträge (plone.app.registry)."
+msgstr "Protokolliert Änderungen der Konfigurationseinträge (Registry)."
```
[the remark this answers is not in the document]
Comment (jensens) [reply src/collective/fingerpointing/locales/de/LC_MESSAGES/collective.fingerpointing.po]: Ack, but plone,.app.registry in brackets is very technical... So ist usually "Die Registry" oder "Die Konfigurationseinträge" (as main wording).

File: src/collective/fingerpointing/locales/de/LC_MESSAGES/collective.fingerpointing.po (hunk 1 above)
Comment (jensens) [reply src/collective/fingerpointing/locales/de/LC_MESSAGES/collective.fingerpointing.po]: ack, I will change it.

File: src/collective/fingerpointing/locales/de/LC_MESSAGES/collective.fingerpointing.po (hunk 2 above)
Comment (jensens) [reply src/collective/fingerpointing/locales/de/LC_MESSAGES/collective.fingerpointing.po]: ack, I will change it.

**Pull request merged**